code/modules/projectiles/projectile/energy/ebow.dm

```diff
@@ -1,16 +1,25 @@
 /obj/item/projectile/energy/bolt //ebow bolts
 	name = "bolt"
 	icon_state = "cbbolt"
-	damage = 8
+	damage = 15
 	damage_type = TOX
 	nodamage = 0
 	knockdown = 100

```

Comment:
Still knock down stuns

Reply:
intended, those are the traitor ones.

Reply:
`Nerfs large ebows, makes traitor ebows do heavy radiation rather than knockout`
You said *rather* so that was my mix up sorry